noir/noir-repo/acvm-repo/acvm_js/src/execute.rs

```diff
@@ -113,6 +113,9 @@ pub async fn execute_circuit_with_black_box_solver(
 
                 acvm.resolve_pending_foreign_call(result);
             }
+            ACVMStatus::RequiresAcirCall(_) => {
+                todo!("Handle acir calls in acvm JS");
```

Comment:
How difficult would this be? It seems like a potential gotcha to users using folding with acvm js

Reply:
I'm working on it now. But as you said this is not ready to be documented so we shouldn't see anyone using folding yet.